src/test/crimson/seastar_runner.h

```diff
@@ -71,16 +71,22 @@ struct SeastarRunner {
     auto ret = app.run(argc, argv, [this] {
       on_end.reset(new seastar::readable_eventfd);
       return seastar::now().then([this] {
-	begin_signaled = true;
-	[[maybe_unused]] auto r = ::eventfd_write(begin_fd.get(), APP_RUNNING);
-	assert(r == 0);
-	return seastar::now();
+#ifdef __aarch64__
+        seastar::smp::invoke_on_all([] {
+          using namespace std::chrono;
+          eastar::engine().update_blocked_reactor_notify_ms(duration_cast<milliseconds>(10000h));
```

Comment:
s/eastar/seastar/

does this compile? also this deserves a comment explaining why update the notify timeout and why 10000h.